test/powershell/engine/Cdxml/Cdxml.Tests.ps1

```diff
@@ -55,7 +55,12 @@ Describe "Cdxml cmdlets are supported" -Tag CI,RequireAdminOnWindows {
         # create the class and instances
         # and track the exitcode for the compilation of the mof file
         # if there's a problem, there's no reason to keep going
-        $result = MofComp.exe ${script:createMof}
+        $testMof = Get-Content -Path ${script:createmof} -Raw
+        $currentTimeZone = [System.TimeZoneInfo]::Local
+        $UTCOffset = ($currentTimeZone.GetUtcOffset([datetime]::new(2008, 01, 01, 0, 0, 0))).TotalMinutes.ToString()

```

Comment:
I was wondering if the 1/1/2008 date was special (since I hadn't got to the MOF change). Perhaps a comment indicating that this date is reflected in the MOF file.

Reply:
I'll add a comment that this date is not special but matches what is in the mof